ouroboros-consensus/src/Ouroboros/Consensus/Protocol/PBFT.hs

```diff
@@ -132,7 +133,7 @@ data PBftParams = PBftParams {
 
       -- | Signature threshold. This represents the proportion of blocks in a
       -- pbftSignatureWindow-sized window which may be signed by any single key.
-    , pbftSignatureThreshold :: Double
+    , pbftSignatureThreshold :: PBftSignatureThreshold
```

Comment:
This is an unrelated change in core consensus code. Leave it out of this PR.

Reply:
Done!